test/integration/examples-good/dune

```diff
@@ -18,4 +18,4 @@
 ;;
 ;; (alias
 ;;  (name runtest)
-;;  (action (diff cpp.expected cpp.output)))
\ No newline at end of file
+;;  (action (diff cpp.expected cpp.output)))
```

Comment:
Am I right in thinking that this doesn't do anything yet? As far as I could tell the cpp output is not yet being tracked, correct?

Reply:
Yes that's right - `;;` is line comment.